fineract-provider/dependencies.gradle

```diff
@@ -30,7 +30,8 @@ dependencies {
     //
     implementation(
             //'ch.vorburger.mariaDB4j:mariaDB4j:2.4.0',
-
+            'org.springframework.boot:spring-boot-starter-security',
+            'org.springframework.security.oauth.boot:spring-security-oauth2-autoconfigure:2.1.6.RELEASE',
```

Comment:
We are trying to avoid specifying version numbers here.
Set the dependency and the version number in build.gradle. That way this is free of version numbers